client/components/App.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+
+const App = props => {
+    return (
```

Comment:
I'm guessing that these are placeholders?

Reply:
I wanted to render something to the screen, this is most definitely a placeholder